frontend/src/component/project/NewProjectCard/ProjectOwners/ProjectOwners.tsx

```diff
@@ -41,23 +41,21 @@ const useOwnersMap = () => {
     };
 };
 
-const StyledContainer = styled('div')(({ theme }) => ({
-    marginBottom: theme.spacing(1),
-    display: 'flex',
-    alignItems: 'flex-end',
-}));
-
 const StyledUserName = styled('p')(({ theme }) => ({
     fontSize: theme.typography.body1.fontSize,
     margin: theme.spacing(0, 0, 0.5, 0),
+    overflowX: 'hidden',
+    textOverflow: 'ellipsis',
+    textWrap: 'nowrap',
+    alignSelf: 'end',
 }));
 
 export const ProjectOwners: FC<IProjectOwnersProps> = ({ owners = [] }) => {
     const ownersMap = useOwnersMap();
     const users = owners.map(ownersMap);
 
     return (
-        <StyledContainer>
+        <>
```

Comment:
Why is empty wrapper needed?

Reply:
Because you can't return multiple elements from a component, so you need a wrapper. We want these elements to flow into the wrapping grid container instead of being in their own little box. This makes it so that they're part of the same grid instead of a nested box.

Reply:
Ah, but you're right! There aren't multiple items. I'll fix.

Reply:
No, wait, github was tricking me with the fold. There *are* two items, so they need to be wrapped like this.